nuitka/build/static_src/CompiledCodeHelpers.c

```diff
@@ -541,6 +541,44 @@ PyObject *BUILTIN_LEN(PyObject *value) {
     return PyInt_FromSsize_t(res);
 }
 
+PyObject *BUILTIN_ANY(PyObject *value) {
+    CHECK_OBJECT(value);
+
+    PyObject *it = PyObject_GetIter(value);
+    PyObject *(*iternext)(PyObject *);
```

Comment:
This ought to be `iternextfunc` and push down declarations to their first use, i.e. `iternextfunc = Py_TYPE(it)->tp_iternext"